config/routes.rb

```diff
@@ -19,6 +19,7 @@
   delete '/pets/:id', to: 'pets#destroy'
 
   patch '/favorite/:pet_id', to: 'favorite#update'
+  delete '/favorite/:pet_id', to: 'favorite#delete'
```

Comment:
I think the restful method name is destroy right?

Reply:
Changed to destroy